enhancements/management-workload-partitioning.md

```diff
@@ -0,0 +1,787 @@
+will use and to which we contribute, and make the changes to add the
+labels ourselves, starting with
+
+* performance-addon-operator
+* sriov-network-operator
+* ptp-operator
+* local-storage-operator
+* cluster-logging (fluentd)
+* klusterlet and klusterlet add-ons (RHACM)
+* kubevirt
+* N3000
+
+Due to time constraints, the first iteration of this work will also
+require the
+[performance-addon-operators](https://github.com/openshift-kni/performance-addon-operators)
+to configure the [kube-reserved and system-reserved CPU sets for
+kubelet](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/)
+and enable the static CPU manager. We expect those values to match the
+CPU set used for management workloads. Future work will eliminate the
+need for the extra operator (see GA graduation criteria).
+
+### User Stories
+
+#### Radio Access Network (RAN) Use Case
+
+In the context of telcocommunications service providers' 5G Radio
+Access Networks, it is increasingly common to see "cloud native"
+implementations of the 5G Distributed Unit (DU) component. Due to
+latency constraints, this DU component needs to be deployed very close
+to the radio antenna for which it is responsible. In practice, this
+can mean running this component on anything from a single server at
+the base of a remote cell tower or in a datacenter-like environment
+serving several base stations.
+
+A hypothetical DU example is an unusually resource-intensive workload,
+requiring 20 dedicated cores, 24 GiB of RAM consumed as huge pages,
+multiple SR-IOV NICs carrying several Gbps of traffic each, and
+specialized accelerator devices. The node hosting this workload must
+run a realtime kernel, be carefully tuned to ensure low-latency
+requirements can be met, and be configured to support features like
+Precision Timing Protocol (PTP).
+
+The most constrained resource for RAN deployments is CPU, so for now
+the focus of this enhancement is managing CPU requirements.
+Kubernetes resource requests for CPU time are measured in fractional
+"[Kubernetes
+CPUs](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu)". For
+bare metal, 1 Kubernetes CPU is equivalent to 1 hyperthread.
+
+Due to the resource-intensive workload, the overhead for platform
+components such as the OpenShift platform and add-on operators is
+severely restricted when compared to other uses. For example, one
+customer has allocated 4 hyperthreads for all components other than
+their own workloads. OpenShift currently requires 7.
+
+### Implementation Details/Notes/Constraints
+
+#### High-level End-to-end Workflow
+
+This section outlines an end-to-end workflow for deploying a cluster
+with workload partitioning enabled and how pods are correctly
+scheduled to run on the management CPU pool.
+
+1. User sits down at their computer.
+2. The user creates a machine config manifest to configure CRI-O to
```

Comment:
We need to show the manifests somewhere below in the details/implentation sections.